mllib/src/test/scala/org/apache/spark/mllib/linalg/distributed/RowMatrixSuite.scala

```diff
@@ -121,6 +122,20 @@ class RowMatrixSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(objectBiggerThanResultSize.getMessage.contains("it's bigger than maxResultSize"))
   }
 
+  test("getTreeAggregateIdealDepth with unlimited driver size") {
```

Comment:
nit but I would add a JIRA prefix like `SPARK-33043: getTree ..` in the test name.